node/src/components/storage/lmdb_ext.rs

```diff
@@ -40,6 +40,10 @@ pub enum LmdbExtError {
     Other(Box<dyn std::error::Error + Send + Sync>),
 }
 
+#[derive(Debug, Error)]
+#[error("{0}")]
+pub struct BytesreprError(pub bytesrepr::Error);
```

Comment:
Is there an option to have a "std" feature on `cargo-types` that implements `Error` on `bytesrepr::Error`, instead of adding this extra wrapper here?

Reply:
I very strongly don't want to provide that functionality.  `casper-types` is meant to be consumed by smart-contracts, and I don't want to make it easy for those to accidentally include the std lib again.